lib/gcloud/dns/record/list.rb

```diff
@@ -45,6 +45,18 @@ def next
         end
 
         ##
+        # Retrieve all records and add to the current list.
+        # This changes the contained records.
```

Comment:
Agreed. I'm struggling on how to best explain it. But at this point I don't know if this is the public API or the private API...